app/client/src/ce/sagas/helpers.ts

```
@@ -96,3 +97,16 @@ export const createDatasourceAPIPayloadFromAction = (
+// This function is extended in EE for UI modules
+export function* getLayoutSavePayload(
+  widgets: FlattenedDSL<unknown>,
+  editorConfigs: Record<string, unknown>,
+) {
+  const nestedDSL = nestDSL(widgets, Object.keys(widgets)[0]);
+
+  return {
+    ...editorConfigs,
+    dsl: nestedDSL,
+  };
+}
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)